lib/hooks/useLoggedInUser.js

```diff
@@ -0,0 +1,3 @@
+import { useLoggedInUser } from '../../components/UserProvider';
+
```

Comment:
I think it makes sense to import `UserContext` and declare `useLoggedInUser` in this file.
In the future, when we need to refactor the hook, we can do it here.

```suggestion
import { UserContext } from '../../components/UserProvider';

const useLoggedInUser = () => React.useContext(UserContext);

```